doc/content/source/indicators/ElementSizeIndicator.md

```diff
@@ -0,0 +1,32 @@
+# ElementSizeIndicator
+
+!syntax description /Adaptivity/Indicators/ElementSizeIndicator
+
+## Description
+
+This indicator takes `element_size` MaterialProperty as an input and directly adapts
+the mesh element size to be equal to the `element_size`. Enough number of `initial_steps`
+  in the `Adaptivity` block should be provided so that the adapted element size
+equals the given `element_size`.
```

Comment:
This text implies that the indicator enforces the mesh size automatically, which they don't. This indicator just calculates the element size from a material property. Also the mesh size is not adapted to be equal to the `element_size`. The minimum element size marker, when used, will keep refining until the min element size is reached. 

Please change this text accordingly.